tiatoolbox/utils/misc.py

```diff
@@ -712,6 +712,50 @@ def string_to_tuple(in_str):
     return tuple(substring.strip() for substring in in_str.split(","))
 
 
+def prepare_file_dir_cli(img_input, output_path, file_types, mode, sub_dirname):
+    """Prepares cli for running code on multiple files or a directory.
```

Comment:
cli should be CLI as it is an acronym for command line interface.

Reply:
resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/220/commits/625222fdb85414b351af4f908c31cec021d1307b